From 91b64075f6ebbf165642f323ea9776ec9fdd4a60 Mon Sep 17 00:00:00 2001
From: Herman van Hazendonk <github.com@herrie.org>
Date: Fri, 5 Jul 2019 08:40:45 +0200
Subject: [PATCH] Don't handle displayInactive event

Handling only displayOn/displayOff events is enough to manage the display state.

Signed-off-by: Simon Busch <morphis@gravedo.de>
Signed-off-by: Herman van Hazendonk <github.com@herrie.org>
---
Upstream-Status: Inappropriate [LuneOS specific]

 src/pwrevents/suspend.c | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/src/pwrevents/suspend.c b/src/pwrevents/suspend.c
index 4b1bb90..e3b4e6a 100644
--- a/src/pwrevents/suspend.c
+++ b/src/pwrevents/suspend.c
@@ -923,8 +923,7 @@ DisplayStatusCb(LSHandle *handle, LSMessage *message, void *user_data)
         event = json_object_get_string(event_obj);
         if (strncmp(event, "displayOn", 9) == 0)
             gDisplayIsOn = true;
-        else if (strncmp(event, "displayOff", 10) == 0 ||
-                 strncmp(event, "displayInactive", 15) == 0)
+        else if (strncmp(event, "displayOff", 10) == 0)
             gDisplayIsOn = false;
     }
 
